src/UI/Components/Diagram/Canvas.test.tsx

```diff
@@ -747,6 +747,7 @@ describe("Canvas.tsx", () => {
   it("creates valid object for order_api when creating entity from scratch", () => {});
   it("creates valid object for order_api when editing entity", () => {});
 
+  it("WHEN strict_modifier_enforcement is set to false for some services THEN for only these instances and their connections editing/deleting is blocked", () => {});
```

Comment:
can you make a ticket for those tests so we don't forget about them?

Reply:
Done, added to New and to Epic